src/components/Transactions/Transfer/TransferAddForm.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+
+import stripesForm from '@folio/stripes/form';
+import {
+  Button,
+  Col,
+  Modal,
+  ModalFooter,
+  Row,
+  TextArea,
+  TextField,
+} from '@folio/stripes/components';
+import {
+  FieldSelect,
+  FieldTags,
+  validateRequired,
+} from '@folio/stripes-acq-components';
+
+const TRANSFER_FORM = 'transferForm';
+
+const footer = (onClose, onSave) => (
+  <ModalFooter>
+    <Button
+      buttonStyle="primary"
+      data-test-add-transfer-save
+      onClick={onSave}
+    >
+      <FormattedMessage id="ui-finance.transactions.button.confirm" />
+    </Button>
+    <Button
+      data-test-add-transfer-cancel
+      onClick={onClose}
+    >
+      <FormattedMessage id="ui-finance.transactions.button.cancel" />
+    </Button>
+  </ModalFooter>
+);
+
+const TransferAddModal = ({ handleSubmit, onClose, funds }) => (
+  <Modal
+    id="add-transfer-modal"
+    label={<FormattedMessage id="ui-finance.transactions.transfer.title" />}
+    footer={footer(onClose, handleSubmit)}
```

Comment:
I'm not sure if this is the best way to write, cc @NikitaSedyx

Reply:
agreed, always ned component

Reply:
plus name convention

Reply:
fixed